.github/workflows/documentation.yml

```
@@ -0,0 +1,26 @@
+name: Documentation
+
+on:
+  push:
+    branches:
+      - master
+    tags: '*'
+  pull_request:
+
+jobs:
+  build:
```

Sugficient permissopns for the tolen (see eg https://documenter.juliadocs.org/dev/man/hosting/#GitHub-Actions):
```suggestion
  build:
    permissions:
      contents: write
      statuses: write
```